zkevm-circuits/src/evm_circuit/execution/call.rs

```diff
@@ -91,7 +91,7 @@ impl<F: Field> ExecutionGadget<F> for CallGadget<F> {
         let tx_id = cb.call_context(None, CallContextFieldTag::TxId);
         let mut reversion_info = cb.reversion_info(None);
         let [caller_address, is_static, depth] = [
```

Comment:
Do you need to update the name of this variable as well, or is it the case that the caller_address in this context is the callee_address of the previous context?

Reply:
> is it the case that the caller_address in this context is the callee_address of the previous context?

Yes.

Perhaps we should use `current_address` here.